app/components/nav-bar.js

```diff
@@ -9,6 +9,17 @@ export default class NavBar extends Component {
     return !(String(this.session.currentRouteName).includes('public'));
   }
 
+    @computed('session.currentRouteName')
+  get isNotWizardPage() {
+    const wizardPages = String(this.session.currentRouteName).includes('events.view.edit.basic-details')
+    || String(this.session.currentRouteName).includes('events.view.edit.attendee')
+    || String(this.session.currentRouteName).includes('events.view.edit.sessions-speakers')
+    || String(this.session.currentRouteName).includes('events.view.edit.index')
+    || String(this.session.currentRouteName).includes('events.view.edit.sponsors')
+    || String(this.session.currentRouteName).includes('events.view.edit.other-details');
```

Comment:
What all these strings have in common?

Reply:
so, if i do `events.view.edit`, the page that opens up after `Manage Events` also doesn't have search box. Do we want that??

Reply:
Yes